go/indexer/db/db.go

```diff
@@ -772,19 +557,41 @@ func (d *Database) GetWithdrawalsByAddress(address l2common.Address, page Pagina
 		withdrawals[i].Batch = batch
 	}
 
-	return withdrawals, nil
-}
+	const selectWithdrawalCountStatement = `
```

Comment:
I'd pull these constants out of the method definition. Will avoid them being recreated on every method call.